### PR TITLE
feat(api): add generated clientset & fake clients

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham14bajpai@gmail.com>

**Why is this PR required? What issue does it fix?**:
_Fixes #162 

**What this PR does?**:
This PR adds generated clientsets and fake clients for jiva CRs for mock tests. This is required for mock testing upgrade, openebsctl and other components which might use jiva CRs in future.

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** : 
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`. 
- [ ] Has the change log section been updated? 
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Does this PR change require updating Helm Chart? If yes, mention the Helm Chart PR #<PR number>
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track: 
- [ ] (Optional) If documentation changes are required, which issue  https://github.com/openebs/website is used to track them:

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
Did you used https://github.com/kubernetes/code-generator/ for generating this?

I was trying to do it for some time and it was not able to generate the `client/versioned/typed` files correctly. Just curious.

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ set -o pipefail
   # so we can install the tools.
   #cd $(dirname "${0}")
   cd vendor/k8s.io/code-generator/
```
Comment (shubham14bajpai) [inline hack/update-codegen.sh:45]: @Abhishek-kumar09 Yes you can check out this shell script. Also added the `// +genclient` tag to the api structs for code-generator to identify the CR's go definition

**Review by vharsh — APPROVED** [review]
lgtm

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**